templates/module/csharp/.tizen.tmpl/tizen-manifest.xml.tmpl

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest package="{{tizenIdentifier}}" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
+<manifest package="{{tizenIdentifier}}" version="1.0.0" api-version="6.5" xmlns="http://tizen.org/ns/packages">
```

Comment:
We assume that .NET modules are built for Tizen 6.5+ by default (so that an embedder with the `_nui` suffix is used whenever applicable). If you want to use a .NET module on Tizen 6.0 or below, you can change this behavior by manually changing the `api-version` value in `.tizen/tizen-manifest.xml`.